resources/views/projects/create.blade.php

```diff
@@ -68,60 +59,58 @@
 			</div>
 		</div> --}}
 		<!-- Associated Course -->
-		<div class="field">
+		<div class="field {{ $errors->has('courses') ? 'error': '' }}">
 			<label for="courses">Associated Course</label>
 			<select name="courses" class="ui search dropdown">
 				<option value="">Web Development Class</option>
 				@if(isset($courses))
-					@foreach($courses -> all() as $course)
-						<option value={{$course['id']}}>
-						{{$course['name']}}</option>
+					@foreach($courses->all() as $course)
+						<option value={{ $course->id }}>{{ $course->name }}</option>
 					@endforeach
 				@endif
 			</select>
 		</div>
 		<!-- Labels -->
-		<div class="field">
+		<div class="field {{ $errors->has('label') ? 'error': '' }}">
 			<label for="label">Labels</label>
-			<select name="label[]" multiple="" class="ui fluid dropdown">
+			<select name="label" class="ui search dropdown">
```

Comment:
should this be `label[]`? No recuerdo si quedamos en que había que modificarlo aquí o de otra manera por el problema que se generó al hacer el cambio en el archivo registerProject.blade.php -> create.blade.php

Thoughts? @hecerinc @anakarenbm

Reply:
Creo que habiamos quedado en que deberia de ser simplemente "Label", pero mejor que nos aseguren @anakarenbm y @hecerinc